### ISSUE TITLE
When the button is first drawn it has a double height top line

### ISSUE THREAD

**Issue by barry-irvine** [google-pay/compose-pay-button#86]
When the button is first being initialised (before the card number is added etc). it has a weird double line artifact at the top.

I'm using version 1.0.0

![image](https://github.com/user-attachments/assets/068b4c77-a051-4cca-b306-282f2d4d8a40)

**Comment by dmengelt** [comment]
@barry-irvine This is supposed to be a loading indicator 😉

![google-pay-button-loading-indicator](https://github.com/user-attachments/assets/ae438f3e-56f5-4843-90ae-69bbabd14ec9)

**Comment by barry-irvine** [comment]
Yeah, I've just realised it's the LinearProgressIndicator when in dark mode! Although I think if the rest of the animations had been as smooth as above then it would have been better anyway. How is yours so smooth?!
